### PR TITLE
Cassandra Integration (commits prior to big refactoring)

### PR DESCRIPTION
**Pull request by igormoreno**
<a href='#crh-start'></a><a href='#crh-data-%7B%22processed%22%3A%20%5B%22https%3A//github.com/benchflow/data-manager/pull/36%23issuecomment-277182273%22%2C%20%22https%3A//github.com/benchflow/data-manager/pull/36%23discussion_r98419892%22%2C%20%22https%3A//github.com/benchflow/data-manager/pull/36%23discussion_r98427352%22%2C%20%22https%3A//github.com/benchflow/data-manager/pull/36%23discussion_r98475779%22%2C%20%22https%3A//github.com/benchflow/data-manager/pull/36%23discussion_r98479340%22%2C%20%22https%3A//github.com/benchflow/data-manager/pull/36%23discussion_r98642039%22%2C%20%22https%3A//github.com/benchflow/data-manager/pull/36%23discussion_r99287150%22%2C%20%22https%3A//github.com/benchflow/data-manager/pull/36%23discussion_r99287174%22%5D%2C%20%22comments%22%3A%20%7B%22General%20Comment%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/benchflow/data-manager/pull/36%23issuecomment-277182273%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22%3A%2B1%3A%3Ca%20href%3D%27%23crh-update-none%27%3E%3C/a%3E%22%2C%20%22created_at%22%3A%20%222017-02-03T07%3A28%3A25Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/7163641%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/VincenzoFerme%22%7D%7D%5D%2C%20%22title%22%3A%20%22General%20Comment%22%7D%2C%20%22Pull%202f568b9933ac0adc7d15be6da6d4ccec0ab5866e%20Dockerfile.test%205%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/benchflow/data-manager/pull/36%23discussion_r98419892%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22Define%20an%20%60ENV%60%20and%20use%20its%20value%20to%20parametrise%20the%20version%20you%20refer%20to.%22%2C%20%22created_at%22%3A%20%222017-01-30T11%3A11%3A34Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/7163641%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/VincenzoFerme%22%7D%7D%2C%20%7B%22body%22%3A%20%22Do%20you%20mean%20something%20like%5Cr%5Cn%60%60%60%5Cr%5CnENV%20DATA_MANAGER_VERSION%20dev%5Cr%5CnADD%20./target/universal/data-manager-%24%7BDATA_MANAGER_VERSION%7D.tgz%20/app/%5Cr%5Cn%60%60%60%5Cr%5Cn%3F%22%2C%20%22created_at%22%3A%20%222017-01-30T12%3A03%3A23Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/229440%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/igormoreno%22%7D%7D%2C%20%7B%22body%22%3A%20%22Yes%22%2C%20%22created_at%22%3A%20%222017-01-30T16%3A12%3A58Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/7163641%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/VincenzoFerme%22%7D%7D%2C%20%7B%22body%22%3A%20%22I%20don%27t%20understand%20very%20well.%20I%20suppose%20this%20version%20is%20something%20like%20a%20tag%20or%20a%20branch%20on%20git%2C%20right%3F%20But%20when%20we%20build%20we%20don%27t%20read%20this%20information%20%28at%20least%20not%20right%20now%29%20to%20generate%20the%20package.%20The%20version%20name%20%28for%20example%20%5C%22dev%5C%22%29%20now%20is%20inside%20the%20sbt%20configuration%20file%22%2C%20%22created_at%22%3A%20%222017-01-30T16%3A26%3A57Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/229440%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/igormoreno%22%7D%7D%2C%20%7B%22body%22%3A%20%22Yes%2C%20that%20is%20fine.%20Defining%20the%20environment%20variable%20in%20the%20Dockerfile%2C%20makes%20it%20available%20in%20the%20containers%20executed%20using%20the%20built%20Dockerfile.%22%2C%20%22created_at%22%3A%20%222017-01-31T10%3A56%3A59Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/7163641%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/VincenzoFerme%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3A%2B1%3A%22%2C%20%22created_at%22%3A%20%222017-02-03T07%3A28%3A09Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/7163641%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/VincenzoFerme%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3A%2B1%3A%3Ca%20href%3D%27%23crh-update-complete%27%3E%3C/a%3E%22%2C%20%22created_at%22%3A%20%222017-02-03T07%3A28%3A25Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/7163641%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/VincenzoFerme%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20Dockerfile.test%3AL4-12%22%7D%7D%7D'></a>
<a href='https://www.codereviewhub.com/'></a>
- [x] <a href='#crh-comment-Pull 2f568b9933ac0adc7d15be6da6d4ccec0ab5866e Dockerfile.test 5'></a> &nbsp;<b><a href='https://github.com/benchflow/data-manager/pull/36#discussion_r98419892'>File: Dockerfile.test:L4-12</a></b>
- <a href='https://github.com/VincenzoFerme'></a> Define an `ENV` and use its value to parametrise the version you refer to.
- <a href='https://github.com/igormoreno'></a> Do you mean something like
```
ENV DATA_MANAGER_VERSION dev
ADD ./target/universal/data-manager-${DATA_MANAGER_VERSION}.tgz /app/
```
?
- <a href='https://github.com/VincenzoFerme'></a> Yes
- <a href='https://github.com/igormoreno'></a> I don't understand very well. I suppose this version is something like a tag or a branch on git, right? But when we build we don't read this information (at least not right now) to generate the package. The version name (for example "dev") now is inside the sbt configuration file
- <a href='https://github.com/VincenzoFerme'></a> Yes, that is fine. Defining the environment variable in the Dockerfile, makes it available in the containers executed using the built Dockerfile.

<a href='https://www.codereviewhub.com/benchflow/data-manager/pull/36?mark_as_completed=1'></a>&nbsp;<a href='https://www.codereviewhub.com/benchflow/data-manager/pull/36?approve=1'></a>&nbsp;<a href='https://github.com/benchflow/data-manager/pull/36'></a>
<a href='#crh-end'></a>

### CODE REVIEW

**Comment by VincenzoFerme** [comment]
This pull request is now managed with CodeReviewHub. <a href='https://github.com/benchflow/data-manager/pull/36'>Refresh</a>.<a href='#crh-update-none'></a>

File: Dockerfile.test [hunk 1]
```diff
@@ -4,9 +4,8 @@ MAINTAINER Vincenzo FERME <info@vincenzoferme.it>
 
 ENV DATA_MANAGER_VERSION v-dev
 
-COPY ./conf/prod.conf /app/
+ADD ./target/universal/data-manager-dev.tgz /app/
```
Comment (VincenzoFerme) [inline Dockerfile.test]: Define an `ENV` and use its value to parametrise the version you refer to.

Comment (igormoreno) [inline Dockerfile.test]: Do you mean something like
```
ENV DATA_MANAGER_VERSION dev
ADD ./target/universal/data-manager-${DATA_MANAGER_VERSION}.tgz /app/
```
?

Comment (VincenzoFerme) [inline Dockerfile.test]: Yes

Comment (igormoreno) [inline Dockerfile.test]: I don't understand very well. I suppose this version is something like a tag or a branch on git, right? But when we build we don't read this information (at least not right now) to generate the package. The version name (for example "dev") now is inside the sbt configuration file

**Comment by VincenzoFerme** [comment]
Yes, that is fine. Defining the environment variable in the Dockerfile, makes it available in the containers executed using the built Dockerfile.

File: Dockerfile.test (hunk 1 above)
Comment (VincenzoFerme) [inline Dockerfile.test]: Yes, that is fine. Defining the environment variable in the Dockerfile, makes it available in the containers executed using the built Dockerfile.

Comment (VincenzoFerme) [inline Dockerfile.test]: :+1:

Comment (VincenzoFerme) [inline Dockerfile.test]: :+1:<a href='#crh-update-complete'></a>

**Comment by VincenzoFerme** [comment]
:+1:<a href='#crh-update-none'></a>

**Pull request merged**